xbmc/utils/DisplayInfo.cpp

```diff
@@ -134,6 +140,21 @@ void CDisplayInfo::LogInfo() const
               m_hdr_static_metadata->desired_content_max_frame_avg_luminance,
               m_hdr_static_metadata->desired_content_max_luminance);
   }
+
+  if (m_colorimetry)
+  {
+    CLog::Log(LOGINFO, "[display-info] supported colorimetry:");
+    CLog::Log(LOGINFO, "[display-info]   xvycc_601:   {}", m_colorimetry->xvycc_601);
```

Comment:
Official names and without underscores? Like `xvYCC BT.601`?

Reply:
I don't know what the official notation is, do you know for all of them?

Reply:
I guess something like that:

xvYCC BT.709
sYCC BT.601
opYCC BT.601 or Adobe YCC BT.601
opRGB or optional RGB or Adobe RGB
BT.2020 cYCC
BT.2020 YCC
BT.2020 RGB
ST2113 RGB
ICtCp

edid-decode has names like this: https://github.com/ValZapod/edid-decode/blob/master/parse-cta-block.cpp#L2072-L2092

Reply:
see CTA-861-G table 71 https://web.archive.org/web/20171201033424/https://standards.cta.tech/kwspub/published_docs/CTA-861-G_FINAL_revised_2017.pdf

eg xvYCC_601 would be close enough

Reply:
BTW: if you don't like the underscore then just drop it, eg "xvYCC 601" but othterwise I strongly suggest to stick to the naming in official standards as closely as possible (eg don't add BT etc) to avoid any ambiguity

Reply:
Interesting, because CEA-861-E actually says "AdobeRGB - optional RGB (opRGB)", so I would say that's official. Similar for AdobeYCC601. But I concur, it's better to drop "BT." prefix.

Reply:
read the first page of the linked pdf (this is the latest CTA-861 version I know of)

> You are being contacted as you have previously obtained a copy CTA‐861‐E, ANSI/CTA‐861‐F or
ANSI/CTA‐861‐G. These documents have been revised from their original published versions to
remove references to Adobe. The company in question requested this change, citing the fact that
its company name is trademarked.

> Users are cautioned that implementation of Adobe RGB (1998) requires compliance with Adobe
RGB (1998) Color Image Encoding, Version 2005‐05, May 2005, Adobe Systems Incorporated. Any
use of the name Adobe RGB must be a reference to the Adobe RGB (1998) color space and color
encodings as defined exclusively by Adobe, and CTA‐861 adopters should not use the Adobe RGB
trademark in connection with their implementations of that standard.